runtime/src/main/java/org/corfudb/runtime/object/PersistenceOptions.java

```diff
@@ -27,4 +27,7 @@ public class PersistenceOptions {
     @Builder.Default
     @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
     Optional<Long> writeBufferSize = Optional.empty();
+
+    @Builder.Default
+    Long reportingFrequency = 5L;
```

Comment:
If metrics are collected every minute this means rocksdb metrics would get reported every 5 minutes right? If so would it help to have a comment here?